docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:20
+
+ARG PORT=3000
+
+ENV NODE_ENV=production
+
+WORKDIR /src
+
+COPY . .
+RUN npm ci
+
+# Generate the prisma client code
+RUN npm install prisma
```

Comment:
Is there a (way to) guarantee that this is the version of prisma specified in package-lock.json? Isn't prisma already installed by the previous line, `npm ci`?

Reply:
Oh no I see npm ci would install @prisma/client, not prisma itself.